pkg/juicefs/mount/builder/common.go

```diff
@@ -99,7 +99,7 @@ func (r *BaseBuilder) genCommonJuicePod(cnGen func() corev1.Container) *corev1.P
 	pod.Spec.Containers[0].Lifecycle = &corev1.Lifecycle{
 		PreStop: &corev1.Handler{
 			Exec: &corev1.ExecAction{Command: []string{"sh", "-c", fmt.Sprintf(
-				"umount %s -l && rmdir %s", r.jfsSetting.MountPath, r.jfsSetting.MountPath)}},
+				"umount %s -l || [ $? -eq 32 ] && rmdir %s || [ $? -eq 1 ]", r.jfsSetting.MountPath, r.jfsSetting.MountPath)}},
```

Comment:
can we 

```
sh -c +e "umount -l xxx; rmdir xxx; exit 0;"
```

to essentially ignore all errors? since prestop hook failures doesn't affect outcome, and we typically don't investigate individual hook errors